docs/cpp/config-clang-mac.md

```diff
@@ -156,25 +159,26 @@ Your new `tasks.json` file should look similar to the JSON below:
             },
             "detail": "Task generated by Debugger."
         }
-    ]
+    ],
+    "version": "2.0.0"
 }
 ```
 
 >**Note**: You can learn more about `tasks.json` variables in the [variables reference](/docs/editor/variables-reference.md).
 
-The `command` setting specifies the program to run. In this case, `"clang++"` is the driver that causes the Clang compiler to expect C++ code and link against the C++ standard library.
+The `command` setting specifies the program to run. In this case, that is`clang++`.
 
-The `args` array specifies the command-line arguments that will be passed to clang++. These arguments must be specified in the order expected by the compiler.
+The `args` array specifies the command-line arguments that are passed to clang++. These arguments must be specified in the order expected by the compiler.
 
-This task tells the C++ compiler to compile the active file (`${file}`), and create an output file (`-o` switch) in the current directory (`${fileDirname}`) with the same name as the active file (`${fileBasenameNoExtension}`), resulting in `helloworld` for our example.
+This task tells the C++ compiler to take the active file (`${file}`), compile it, and create an output file (`-o` switch) in the current directory (`${fileDirname}`) with the same name as the active file but with the `.exe` extension (`${fileBasenameNoExtension}`). This process creates `helloworld`.
```

Comment:
Shouldn't this be:
"with the same name as the active file but without a file extension....

Reply:
I'll update this in a follow up PR

Reply:
Nope, I'll take care of it. I've got a follow up PR in progress fixing a few nits